docs/operating-scylla/nodetool-commands/restore.rst

```diff
@@ -42,7 +49,7 @@ Options
 * ``--prefix`` - The share prefix for object keys of backed up SSTables
 * ``--keyspace`` - Name of the keyspace to load SSTables into
 * ``--table`` - Name of the table to load SSTables into
-* ``--nowait`` - Don't wait on the restore process
+* ``--nowait`` - Returns immediately without waiting for the restore operation to complete.
```

Comment:
```suggestion
* ``--nowait`` - Returns immediately (with task-id) without waiting for the restore operation to complete. The task-id can be used to monitor the process.
```

Reply:
your suggestion repeats what we already have in the added paragraph.

Reply:
> your suggestion repeats what we already have in the added paragraph.

Yes, indeed.
These kind of documents have two meanings:
1. Overview and understanding.
2. References. A quick help when you need it.

The added paragraph is the implementation of 1. The `--nowait` is the implementation of 2.
In general it is OK to repeat the same information in both, for ease of use.

Is it required to be added in this specific case? Debatable, I think it should.